keyboards/clueboard/66/rev4/chconf.h

```diff
@@ -48,7 +48,7 @@
  * @details Frequency of the system timer that drives the system ticks. This
  *          setting also defines the system tick time unit.
  */
-#define CH_CFG_ST_FREQUENCY                 100000
+#define CH_CFG_ST_FREQUENCY                 1000
```

Comment:
We may not want to change this, as it will slow down the board.

Doing so cause serious issues with animations in the planck ez, actually, as well as issues with timing for keys that used any sort of timing (software or otherwise). 

```suggestion
#define CH_CFG_ST_FREQUENCY                 100000
```